merfi/collector.py

```diff
@@ -66,3 +71,36 @@ def debian_release_files(self):
                     if isfile(release_file):
                         result.append(release_file)
         return result
+
+    @property
+    def rpm_repomd_files(self):
```

Comment:
As we discussed earlier, I finally have a branch that implements a collector with `DebRepo` and `RpmRepo` as you described. I'm refactoring that into reviewable chunks now.

We should be able to generalize the collector to find everything - RPMs, RPM repo metadata, Debian repo metadata, vanilla release tarballs, and maybe Docker containers.